cmd/zt_sync_local_blob_test.go

```diff
@@ -82,13 +85,15 @@ func (s *cmdIntegrationSuite) TestSyncUploadWithSingleFile(c *chk.C) {
 }
 
 // regular directory->container sync but destination is empty, so everything has to be transferred
+// this test seems to flake out.
```

Comment:
Does the sleep help with the test being flakey?

Reply:
Yes, quite a bit. It was strange to see.

I don't feel like we're super likely to see whatever was causing this to flake out in a human scenario.